app/representers/api/auth/account_min_representer.rb

```diff
@@ -5,8 +5,7 @@ class Api::Auth::AccountMinRepresenter < Api::Min::AccountRepresenter
   link :stories do
     {
       href: "#{api_authorization_account_stories_path(represented)}#{index_url_params}",
-      templated: true,
-      count: represented.stories.count
```

Comment:
pretty sure nothing relies on this - it was a bit of a "standard" to offer counts, but not a big deal

Reply:
When we someday go through CMS with a fine toothed comb ... would be nice to standardize on what's included in full vs min representers.  Maybe only full get counts and embedded-by-defaults?